packages/babel-node/src/_babel-node.js

```diff
@@ -197,7 +201,7 @@ if (program.eval || program.print) {
     }
 
     // add back on node and concat the sliced args
-    process.argv = ["node"].concat(args);
+    process.argv = ["node", ...args];
```

Comment:
Just a question: Is this for performance? or for readability?

Reply:
Yeah, both of these are O(n). The problem with concat is when it's inside a loop.